Command/SitemapGeneratorCommand.php

```diff
@@ -200,12 +217,21 @@ protected function generateSitemapFromRoutes(Sitemap $sitemap, array $routes, $m
             );
 
             if (empty($routeParameters)) {
-                $sitemap->addItem(
-                    $this->router->generate($routeName, [], true),
-                    $routeConfigurations['lastmod'],
-                    $routeConfigurations['changefreq'],
-                    $routeConfigurations['priority']
-                );
+                $curRouteUrl = $this->router->generate($routeName, [], true);
+
+                if (!$this->isUrlExcluded($curRouteUrl)) {
```

Comment:
You can also remove the else statement by flipping the `if` condition, like this:

```php
if ($this->isUrlExcluded($curRouteUrl)) {
    $this->output->writeln('this URL is excluded by exclude_urls param: ' . $curRouteUrl);
}
```